imports/plugins/core/orders/server/no-meteor/util/getDataForOrderEmail.js

```diff
@@ -136,13 +137,16 @@ export default async function getDataForOrderEmail(context, { order }) {
   }
 
   const copyrightDate = new Date().getFullYear();
-
+  const token = await addAnonymousOrderToken(context, order._id);
```

Comment:
Better to throw an error or omit this from the template data? Is there a use case where the template does not link back to the order at all? A purely headless shop but they do send emails?

Reply:
Agree, better to just omit the data. Possibly a debug-level log explaining why it's omitted.

Reply:
OK I'm going to allow `storefrontUrls` to be missing and proceed anyway. The properties `homepage` and `orderUrl` will be null in that case, but I think it's a valid headless use case and I'm not going to throw an error or log a warning (because it would get logged on every order email). Will check for `":token"` before calling `addAnonymousOrderToken`.